src/components/Books.module.css

```diff
@@ -1,4 +1,81 @@
 .book {
   display: flex;
   flex-direction: column;
+  width: 75rem;
+  height: 10.625rem;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
```

Comment:
- [ ] It would be nice if you could make the wrapper for the books to be in the center. Currently, the wrapper that holds the books and the form is aligned to the left. According to the original design, things are centered on the screen. Please work on your styles to match that 👍 

#### See the original design and the area I marked in green so you can spot what I mean 
![image](https://user-images.githubusercontent.com/50941074/194717671-fddf62e3-4182-415b-8d6e-9658ae025ca9.png)

#### Your Design
![image](https://user-images.githubusercontent.com/50941074/194717854-666976a9-0f2f-4411-8860-b3495ecc3632.png)

- [ ] On the designs above, please note the points below you have to fix in order for your design to match the original design:

1. Center the wrapper that holds all the books so things are placed in the center of the screen. You are having too many white spaces on the right side. 
2. Kindly add a User icon to the right of the navbar just as it is in the original design 👍 
3. Reduce the spacing from the percentage sign to the completed text below it. Please take a look at the original design, there is not much spacing there.
4. Kindly make it so that the input field starts from the left of the screen and the Add Book button aligns to the right together with all the books. Your design is having spacing from the left of the title input field. Please work on it 👍 

![image](https://user-images.githubusercontent.com/50941074/194718133-f3066865-5bb6-4953-9c16-11a27934a0f7.png)